schemas/context/poi-interaction.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/poi-interaction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Point of Interest Interaction",
+  "type": "object",
+  "description": "A Point or Place of Interest(POI) interaction. Used to capture triggered engagement activities when users enter, leave of come within range of a POI.",
```

Comment:
Why does a point of interest have to be directly connected to engagement?  There are many other reasons/use cases for storing a point/place of interest.

Reply:
I still believe that a POI should be completely standalone and not directly connected to interactions, events, etc.

Reply:
ExperienceEvent is about the interaction and the surrounding circumstances so they need to be related at this point of usage. The POI details can be used outside of the interaction. It is not very tightly bound. I do expect that we will see a case where poi-detail is used in a standalone table for management of POIs and I think this does support that.